dev_environment/Vagrantfile

```diff
@@ -7,84 +7,118 @@ cfg = YAML.load_file(File.join(
   File.dirname(File.expand_path(__FILE__)),
   "config.yaml"))['configs']
 
+def nat(config)
+  config.vm.provider "virtualbox" do |v|
+    v.customize ["modifyvm", :id, "--nic2", "natnetwork", "--nat-network2", "NATNetwork101"]
+  end
+end
+
 Vagrant.configure("2") do |config|
-  config.vm.box = "ubuntu/focal64"
+  config.vm.define "l3af" do |l3af|
 
-  # Set boot timeout value
-  config.vm.boot_timeout = 600
+    # Set boot timeout value
+    l3af.vm.boot_timeout = 600
 
-  config.vm.network "forwarded_port", guest: 8080, host: cfg['host_http_port1']
-  config.vm.network "forwarded_port", guest: 8081, host: cfg['host_http_port2']
-  config.vm.network "forwarded_port", guest: 3000, host: cfg['host_grafana_port']
-  config.vm.network "forwarded_port", guest: 9090, host: cfg['host_prometheus_port']
-  config.vm.network "forwarded_port", guest: 7080, host: cfg['host_l3af_config_port']
-  config.vm.network "forwarded_port", guest: 8899, host: cfg['host_l3af_debug_port']
+    l3af.vm.network "forwarded_port", guest: 8080, host: cfg['host_http_port1']
+    l3af.vm.network "forwarded_port", guest: 8081, host: cfg['host_http_port2']
+    l3af.vm.network "forwarded_port", guest: 3000, host: cfg['host_grafana_port']
+    l3af.vm.network "forwarded_port", guest: 9090, host: cfg['host_prometheus_port']
+    l3af.vm.network "forwarded_port", guest: 7080, host: cfg['host_l3af_config_port']
+    l3af.vm.network "forwarded_port", guest: 8899, host: cfg['host_l3af_debug_port']
 
-  config.vm.provider "virtualbox" do |vb|
-    vb.memory = "2048"
-  end
+    l3af.vm.provider "virtualbox" do |vb|
+      vb.memory = "2048"
+    end
+
+    l3af.vm.box = "ubuntu/focal64"
+    if cfg['traffic_mirroring'] == "true"
+      l3af.vm.network :private_network, ip: "192.168.10.40", :netmask => "255.255.255.0"
```

Comment:
Just curious, why IPv4 instead of IPv6?

Reply:
Is there a github issue number tracking that?

Reply:
https://github.com/l3af-project/eBPF-Package-Repository/issues/43